lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -93,6 +95,17 @@ class DetailsRenderer {
    * @param {!DetailsRenderer.DetailsJSON} text
    * @return {!Element}
    */
+  _renderLink(text) {
+    const snippet = text.text || '';
+    const element = this._dom.convertMarkdownLinkSnippets(snippet);
```

Comment:
rather than using markdown here, what about explicitly using `text` for the content and `url` for the URL? 

We originally were avoiding using the markdown conversion for the content that would be coming from the user's site (it was restricted to just the descriptions in the config, helpText, etc), so I don't know if we want to open that door now. If we do go the markdown route, we might as well add the functionality the `text` type and call it a day instead :)

Reply:
Just want to clarify before proceeding....

You want something like: `_renderLink(text, url)` ?

Reply:
well we really just want this to be in details, the variable you've named `text` here is really `details` so it would be something like

```
_renderLink(details) {
  const snippet = details.text || '';
  const url = details.url;
  ...
}
```

Usage: `detailsRenderer.render({type: 'link', text: 'Learn more', url: 'https://developers.google.com/web/...'})`

Reply:
Ah, gotcha. Ok...that makes sense.